src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -14,7 +14,7 @@ import { CommandListArgs } from '../list/list.js';
 
 const SHEET_MIN_X = MapSheet.origin.x + 4 * MapSheet.width; // The minimum x coordinate of a valid sheet / tile
```

Comment:
this logic needs to be scrapped, we need to allow tiles outside the "valid" range of topo50 mapsheets.

Reply:
I thought in the meeting we decided to do this later when the need arises. I double-checked with Rebecca yesterday and she would like to keep the logic in for checking imagery is a valid topo50 mapsheet.

Reply:
I don't know how hard it would be to raise an alert if it's not a valid topo50 mapsheet but still allow it?